chevah_build

```diff
@@ -19,6 +19,12 @@ LIBEDIT_VERSION="20160903-3.1"
 GMP_VERSION="6.1.2"
 SQLITE_VERSION="3.17.0"
 PYSQLITE_VERSION="2.6.3"
+# These are used by the patched get-pip.py.
+# PIP_VERSION is already used by get-pip for something else.
+# For now, we can't use version 8 of pip as our PyPi server does not support
+# simplified names.
+export CHEVAH_PIP_VERSION="7.1.2"
```

Comment:
For uniformity, there's a line further down for exporting needed vars. It is prepended by _# Export the variables needed by the chevahbs scripts and the test phase._